pkg/template/template.go

```diff
@@ -40,20 +41,49 @@ func (p *Processor) Process(template *api.Template) (*configapi.Config, fielderr
 	}
 
 	for i, item := range template.Objects {
+		if obj, ok := item.(*runtime.Unknown); ok {
```

Comment:
We should be reporting eventually, but not here, no.  I do the loop instead of runtime.DecodeList so I can assign an index to the list.  Maybe runtime.DecodeList should return a ValidationErrorList instead, but will do that upstream and we can refactor here later.